user-interface/0003-WTXN-submit_vega_transaction.md

```diff
@@ -0,0 +1,48 @@
+# Submit Vega transaction
+
+A dapp sends a transaction to a wallet, that wallet then broadcasts the transaction to a network.  Therefore the following is broken up into two steps. The transaction could fail at either. Generally: Once the transaction has gone to the network a user can use block explorer to track the transaction, but some tracking in Dapp or wallet will help.
+
+When submitting a Vega transaction of any kind, I...
+
+## Track transaction to wallet
+
+if not connected to a Vega wallet:
+
+- **must** be told that I am not connected and given the [option to connect](0012-WCON-connect_vega_wallet.md) (<a name="0003-WTXN-001" href="#0003-WTXN-001">0003-WTXN-001</a>)
```

Comment:
```suggestion
- **must** be told that I am not connected, so can not submit a transaction, and given the [option to connect](0012-WCON-connect_vega_wallet.md). (Note: this may have happened if the wallet has become disconnected without the user knowing) (<a name="0003-WTXN-001" href="#0003-WTXN-001">0003-WTXN-001</a>)
```